src/nvim/msgpack_rpc/channel.c

```diff
@@ -834,6 +835,7 @@ static void log_client_msg(uint64_t channel_id,
                            msgpack_object msg)
 {
   DLOGN("[msgpack-rpc] client(%" PRIu64 ") -> nvim ", channel_id);
+  log_lock();
```

Comment:
Are these msgpack loggers removed if `DISABLE_LOG` is defined? Won't it cause potential performance issues if many clients are fighting over the same log mutex?